Formula/libtorrent-rasterbar.rb

```diff
@@ -1,8 +1,8 @@
 class LibtorrentRasterbar < Formula
-  desc "C++ bittorrent library by Rasterbar Software"
+  desc "C++ bittorrent library with Python bindings"
```

Comment:
Why did the description change?

Reply:
It's in PR and commit message

Reply:
Sorry I missed that.

```suggestion
  desc "C++ BitTorrent library and client, including popular extensions"
```

I think we should keep the description as close to upstream as possible. I've adapted this from the first sentence of the first paragraph in the README.md. I don't think we need to specifically mention python, many libraries ship python bindings.

Reply:
FYI, I'm open to other descriptions and I didn't check if this triggers audit/style for being too long. But I *do* think there's no need to explicitly mention python.

Reply:
I'm afraid that wording change is not useful, it has a demo client at best and popular extensions relates to BitTorrent protocol.

The reason to mention Python bindings is due to libtorrent often shipping as two separate libraries, `libtorrent-rasterbar` and `python-libtorrent` so to ensure no confusion it would be good to mention. For example when looking to install for Deluge it was not obvious that it definitely included the Python bindings, only the dependency gave a clue.

Ideally it would be separated into two but requires more time to implement.

Reply:
OK, I guess I'm OK with this...